e2e/testcases/reconciler_manager_test.go

```diff
@@ -207,7 +207,7 @@ func validateRootSyncDependencies(nt *nomostest.NT, rsName string) []client.Obje
 	// only happens when upgrading from a very old unsupported version.
 
 	rootSyncCRB := &rbacv1.ClusterRoleBinding{}
-	setNN(rootSyncCRB, client.ObjectKey{Name: controllers.RootSyncPermissionsName(rootSyncReconciler.Name)})
+	setNN(rootSyncCRB, client.ObjectKey{Name: controllers.RootSyncLegacyClusterRoleBindingName})
```

Comment:
another case where setNN is unnecessary abstraction, since it's just setting the name.

Reply:
Removed